### PR TITLE
Actionlib python scripts

### PR DESCRIPTION
**Pull request by scpeters**
These python scripts were on the wiki page but not in this repository. I removed the `roslib` references as suggested in #13.

### CODE REVIEW

File: actionlib_tutorials/scripts/fibonacci_client.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # Prints out the result of executing the action
+    return client.get_result()  # A FibonacciResult
+
+if __name__ == '__main__':
+    try:
+        # Initializes a rospy node so that the SimpleActionClient can
+        # publish and subscribe over ROS.
+        rospy.init_node('fibonacci_client_py')
+        result = fibonacci_client()
+        print "Result:", ', '.join([str(n) for n in result.sequence])
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_client.py]: For Python 3 compatibility this needs parenthesis.

Same below.

File: actionlib_tutorials/scripts/fibonacci_client.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    except rospy.ROSInterruptException:
+        print "program interrupted before completion"
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_client.py]: Error messages should go to stderr.

File: actionlib_tutorials/scripts/fibonacci_server.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import rospy
+
+import actionlib
+
+import actionlib_tutorials.msg
+
+class FibonacciAction(object):
+    # create messages that are used to publish feedback/result
+    _feedback = actionlib_tutorials.msg.FibonacciFeedback()
+    _result   = actionlib_tutorials.msg.FibonacciResult()
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_server.py]: Please avoid aligning the equal signs.

File: actionlib_tutorials/scripts/fibonacci_server.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        # append the seeds for the fibonacci sequence
+        self._feedback.sequence = []
+        self._feedback.sequence.append(0)
+        self._feedback.sequence.append(1)
+        
+        # publish info to the console for the user
+        rospy.loginfo('%s: Executing, creating fibonacci sequence of order %i with seeds %i, %i' % (self._action_name, goal.order, self._feedback.sequence[0], self._feedback.sequence[1]))
+        
+        # start executing the action
+        for i in xrange(1, goal.order):
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_server.py]: For Python 3 compatibility this should use `range`.

File: actionlib_tutorials/scripts/fibonacci_server.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            r.sleep()
+          
+        if success:
+            self._result.sequence = self._feedback.sequence
+            rospy.loginfo('%s: Succeeded' % self._action_name)
+            self._as.set_succeeded(self._result)
+        
+if __name__ == '__main__':
+    rospy.init_node('fibonacci')
+    FibonacciAction(rospy.get_name())
```
Comment (dirk-thomas) [inline actionlib_tutorials/scripts/fibonacci_server.py]: This instance must assigned to a variable in order to not be potentially garbage collected immediately.

**Comment by scpeters** [comment]
thanks to @k-okada for responding to @dirk-thomas 's comments

**Comment by wjwwood** [comment]
Thanks.

**Pull request merged**